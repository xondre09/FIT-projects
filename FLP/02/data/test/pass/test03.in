S   P R
P x P R
P y Q R
Q y Q R
Q   R L
R x S  
R y S  
R   T R
S   R L
T   U y
U y F L
 xxy
