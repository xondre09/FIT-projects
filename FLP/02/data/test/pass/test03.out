S xxy
 Pxxy
 xPxy
 xxPy
 xxyQ 
 xxRy
 xxS 
 xRx
 xS 
 Rx
 S 
R 
 T 
 Uy
F y
